src/strands/tools/executor.py

```diff
@@ -21,7 +21,7 @@ def run_tools(
     handler: Callable[[ToolUse], ToolResult],
     tool_uses: List[ToolUse],
     event_loop_metrics: EventLoopMetrics,
-    request_state: Any,
+    _request_state: Any,
```

Comment:
We should just remove this parameter I think?  No need to pass it if it's not used